src/content/docs/reference/functions/get_host_categories.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: GET_HOST_CATEGORIES function
+description: Reference docs for the httparchive.fn.GET_HOST_CATEGORIES function
+---
+
+# `GET_HOST_CATEGORIES` function
+
+## Description
+We are happy to release the largest open-source dataset of website categories, featuring 147 million hosts and 31 million domains, making it the most extensive open-source data available in this area.
+
+The `GET_HOST_CATEGORIES` function retrieves the category information of a given hostname. It maps hostnames to their respective categories using a pre-classified dataset. 
+
+## Input
+
+### `input_host`
+The function takes a single parameter: `input_host`, which is a STRING representing the hostname to be classified. The hostname can be a full URL (e.g., `https://store.google.com/product`) or just the host (e.g., `store.google.com`).
+
+**Type:** `STRING`
+
+
+### Output
+- ARRAY of STRUCT containing:
+  - `domain` (STRING): The queried host.
+  - `category_id` (INT64): The category ID of the host.
+  - `full_category` (STRING): The full hierarchical category.
+  - `subcategory` (STRING): The most specific subcategory.
+  - `parent_category` (STRING): The parent category.
+
+### Example Query
+
+<Tabs>
+  <TabItem label="Query">
+
+```sql
+SELECT
+  *
+FROM
+  UNNEST(httparchive.fn.GET_HOST_CATEGORIES('apple.com'))
+```
+   
+  </TabItem>
+  <TabItem label="Results">
+
+```json
+[{
+  "host": "apple.com",
+  "category_id": "528",
+  "full_category": "/Internet \u0026 Telecom/Mobile Phones",
+  "subcategory": "Mobile Phones",
+  "parent_category": "/Internet \u0026 Telecom/"
+}, {
+  "host": "apple.com",
+  "category_id": "129",
+  "full_category": "/Computers \u0026 Electronics/Consumer Electronics",
+  "subcategory": "Consumer Electronics",
+  "parent_category": "/Computers \u0026 Electronics/"
+}]
+```
+  </TabItem>
+</Tabs>
+
+You can also integrate this function into another SQL query by joining its result with your target table. Here is an example:
+
+<Tabs>
+  <TabItem label="Query">
+
+```sql
+SELECT
+  r.url,
+  c.category_id,
+  c.full_category,
+  c.subcategory,
+  c.parent_category
+FROM
+  `httparchive.urls.latest_crux_mobile` AS r,
+  UNNEST(httparchive.fn.GET_HOST_CATEGORIES(r.url)) AS c;
+  ```
+  
+  </TabItem>
+</Tabs>
+
+
+## Methodology
+The classification of hostnames is performed using Chrome's Topics API (currently `chrome5`)  by [Nurullah Demir](https://ndemir.com) and [Yohan Beugin](https://yohan.beugin.org/) using [this repository](https://github.com/yohhaan/httparchive-topics-classification). The taxonomy used is version 2, which consists of 469 categories as defined in the [Topics API taxonomy v2](https://github.com/patcg-individual-drafts/topics/blob/a116e9e404dd96c7793fe6542c53e9a9d93cb75b/taxonomy_v2.md).
```

Comment:
Could we include more info about how recently the `chrome5` model was trained?

Reply:
Yes, it makes sense. It's currently the latest model shipped in Chrome for topics.